js/src/widget_ngl.ts

```diff
@@ -398,6 +399,16 @@ class NGLView extends widgets.DOMWidgetView{
     async handleEmbed(){
         var that = this;
         var ngl_msg_archive = that.model.get("_ngl_msg_archive");
+
+        if (!that.model.get("_ngl_is_initialized")) {
```

Comment:
<img width="1344" alt="image" src="https://github.com/nglviewer/nglview/assets/4451957/d2a177ec-6feb-4f11-a048-0c7ab8c9f863">

Reply:
or check for both `ngl_full_stage_parameters` and `_camera_orientation`?

Reply:
Done! It looks `_ngl_full_stage_parameters` object is already modified in `createStage`, so I've just made this a shallow copy so that changes aren't reflected in the shared state until we deliberately sync them back up. I think this is more robust to future changes than checking if `_ngl_full_stage_parameters` is the value it is modified to.